sites/cablinginstall/config/ads.js

```diff
@@ -88,4 +88,12 @@ module.exports = {
     'load-more': { ...CONTENT, path: path('testing/load-more') },
     reskin: { ...RESKIN, path: path('testing/reskin') },
   },
+  innovators: {
+    lb1: { ...LB, path: path('innovators/lb1') },
```

Comment:
Have the ad units been created for the innovators section?

Reply:
<img width="488" alt="Screen Shot 2019-09-10 at 3 51 24 PM" src="https://user-images.githubusercontent.com/6343242/64645427-e33ce800-d3e2-11e9-8a03-7a392f0301c5.png">